apps/main/src/dao/components/PageAnalytics/CrvStats/index.tsx

```diff
@@ -25,9 +25,7 @@ const CrvStats: React.FC = () => {
     }
   }, [veCrvData.totalCrv, veCrvData.fetchStatus, getVeCrvData, provider])
 
-  const veCrvApr = aprLoading
-    ? 0
-    : calculateApr(veCrvFees.fees[1].feesUsd, Number(veCrvData.totalVeCrv) / 10 ** 18, crv)
+  const veCrvApr = aprLoading ? 0 : calculateApr(veCrvFees.fees[1].feesUsd, veCrvData.totalVeCrv.fromWei(), crv)
```

Comment:
Ideally we should move all `fromWei` operation to the API parsing IMO. That would have 2 advantages:
- separating the business logic from the UI
- only repeating for every response, not for every render

You mentioned that you wanted to keep `bignumber` for eventual extra calculations, and maybe `fromWei` could even keep it a bignumber (although by our tests, Number is precise enough).

Reply:
I was thinking of maybe making toGwei lazy and apply memoization, but the calculation is so small and fast I wonder if it makes any difference at all 🤔